### PR TITLE
Updating RTL documentation

### LINKED ISSUE #11301: Documentation: Localization RTL page unclear
### Environment Information

- **Package version(s)**: 7.64.2
- **Browser and OS versions**: Any

### Please provide a reproduction of the bug in a codepen:

Go to the docs page for [Localization](https://developer.microsoft.com/en-us/fabric#/styles/web/localization). It says to use an rtl stylesheet to enable RTL mode. It seems like this isn't supported now. Please could you update the docs to specify how this should be done now? Adding `dir="rtl"` to the html element doesn't reverse the icons on it's own. Do we need to flip the icons manually using `getRTL`?

### Priorities and help requested:

Are you willing to submit a PR to fix? No

Requested priority: Low

Products/sites affected: (if applicable)

### PR DESCRIPTION
**Pull request by khmakoto**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #11301 
- [x] Include a change request file using `$ yarn change`

#### Description of changes

Our Localization documentation regarding RTL did not reflect the current state of the library. This PR updates it so that it does.

#### Focus areas to test

(optional)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/11351)

### CODE REVIEW

File: apps/fabric-website/src/pages/Styles/LocalizationPage/docs/web/LocalizationRTL.md [hunk 1]
```diff
@@ -1,15 +1,14 @@
-Fabric comes with an alternate CSS file for pages written in right-to-left (RTL) languages, such as Arabic and Hebrew. This reverses the order of columns in the responsive grid, making it easy to create an RTL layout without writing additional templates. Many icons are also reversed, particularly those used for navigation such as arrows.
+Fabric comes with built-in support for pages written in right-to-left (RTL) languages, such as Arabic and Hebrew. Fabric achieves this by reversing the order of columns in the responsive grid, making it easy to create an RTL layout without writing additional templates. In addition to this, Fabric Core specifically also reverses many icons, particularly those used for navigation such as arrows.
```
Comment (ecraig12345) [inline apps/fabric-website/src/pages/Styles/LocalizationPage/docs/web/LocalizationRTL.md]: Probably good to be clearer about what's part of Fabric React vs. Core. Something like this:

```suggestion
Fabric comes with built-in support for pages written in right-to-left (RTL) languages, such as Arabic and Hebrew. 

Each component in Fabric React will automatically reverse the layout of elements within the component.

For pages using Fabric Core's responsive grid, the order of columns is automatically reversed, making it easy to create an RTL layout without writing additional templates. Fabric Core also reverses many icons, particularly those used for navigation such as arrows. (Icons in Fabric React are not automatically reversed.)
```

File: apps/fabric-website/src/pages/Styles/LocalizationPage/docs/web/LocalizationRTL.md [hunk 2]
```diff
@@ -1,15 +1,14 @@
...
 
 <!-- headings get auto-generated IDs usually, and this page has two "Implementation" headings -->
 <h3 id="rtl-implementation">Implementation</h3>
 
-Apply the “dir” attribute to the HTML tag, and reference the RTL version of Fabric.
+Apply the `dir` attribute with the `rtl` value to the HTML tag.
```
Comment (ecraig12345) [inline apps/fabric-website/src/pages/Styles/LocalizationPage/docs/web/LocalizationRTL.md]:
```suggestion
Apply the `dir` attribute with value `rtl` to the HTML tag.
```

**Comment by micahgodbolt** [comment]
@HarryGifford does this update make the RTL docs a bit more clear?

**Comment by HarryGifford** [comment]
@micahgodbolt Yes, it's much clearer now. Thank you!

**Pull request merged**